src/bounds.rs

```diff
@@ -0,0 +1,59 @@
+//! Bounds representing an axis aligned bounding box.
+
+use glam::Vec2;
+
+/// Represents an axis aligned bounding box.
+#[derive(Debug, Clone, Copy)]
+pub struct Bounds {
+    half_extents: Vec2,
+    position: Vec2,
+}
+
+impl Bounds {
```

Comment:
I'm actually following this tutorial <https://www.youtube.com/watch?v=MbWK8bCAU2w> and making a bevy character controller plugin... I thought I'd use impacted, but the concept of bounds wasn't available and it is used in the tutorial. So for my case it's been quite useful to implement a player controller... but I'm not familiar with bvh-arena or broccoli and unsure if they provide something similar already.

If you don't think the bounds struct is in the scope of this library then that's fine, but it's been quite useful for my project currently. But I could move it directly into my project too.

Reply:
I see, thank you for the detailed explanation.
I ended up pushing my character controller to a repo which uses this PR, it might help provide a better example of my use case: <https://github.com/tqwewe/bevy_controller_2d>

I'll remove the bounding box from this PR and move it to a different branch in my fork, and if you think it'd be nice to add let me know and I can make another PR for it.

Reply:
Fixed in https://github.com/tqwewe/impacted/commit/b0f92c85ff3ce6bfa3843d131f0c2d59490deb37